next/tailwind.config.js

```diff
@@ -5,8 +5,25 @@ module.exports = {
     "./components/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        // Generated with https://uicolors.app/
+        wunderpurple: {
```

Comment:
This gives us automatically tailwind classes that we can use to show our colors in the site